packages/web/src/shared/components/form/ComboboxInput.tsx

```diff
@@ -120,31 +66,11 @@ const BaseComboboxInput = forwardRef<HTMLInputElement, BaseComboboxInputProps>(
             ignoreDiacritics(normalizedInputValue.toLowerCase())
           )
         );
-        const noExactMatch = filteredItems.every(
-          (item) => item.label.normalize('NFD') !== normalizedInputValue
-        );
```

Comment:
We're fine to get rid of all this extra logic?

Reply:
Yeah, we don't need to create new elements with a combobox anymore